crepecake/src/main/java/com/google/cloud/tools/crepecake/cache/CacheHelper.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.crepecake.cache;
+
+import java.io.File;
+import java.nio.file.Path;
+
+/** Methods for getting static cache properties. */
+class CacheHelper {
```

Comment:
In general, I think we should try not to use names like `*Helper`, maybe we're not categorizing it's purpose correctly. Is there something we can call this that makes more sense? CacheFiles? or something more specific?